integration_tests/pages/induction/InductionPage.ts

```diff
@@ -0,0 +1,26 @@
+import Page, { PageElement } from '../page'
+
+export default abstract class InductionPage extends Page {
+  constructor(pageId: string, options?: { axeTest?: boolean }) {
+    super(pageId, options)
+  }
+
+  clickBackLinkTo = <T extends Page>(expected: new () => T): T => {
```

Comment:
I've not written any tests currently call to call this method, but if we did want to write a test that clicked the Back link we should be able to do something like:

```
const affectAbilityToWorkPage = Page.verifyOnPage(AffectAbilityToWorkPage)
...
...
const workAndInterestsPage = affectAbilityToWorkPage.clickBackLinkTo(WorkAndInterestsPage)

```